tests/autoyast/installation.pm

```diff
@@ -103,6 +116,10 @@ sub run {
                 wait_idle(5);
                 $num_errors++;
             }
+            if (match_has_tag('linuxrc-install-fail')) {
+                save_logs_in_linuxrc("stage1_error$i");
+                die "installation ends in linuxrc";
```

Comment:
that's a better idea than just recording "fail" as it is more obvious in which step the test ends. Can you call the "save_logs_in_linuxrc" in a post fail hook and just do the die here? The post_fail_hook will be called after the die has recorded the test to fail.